src/dto/service_predict.hpp

```diff
@@ -40,6 +40,7 @@ namespace dd
       DTO_FIELD(String, service);
       DTO_FIELD(Object<Parameters>, parameters) = Parameters::createShared();
       DTO_FIELD(List<String>, data);
+      DTO_FIELD(Boolean, chain) = false;
```

Comment:
"chain" should not be in the DTO since it is added to the APIData internally. We need to find a better way to pass it to the predict call (maybe directly as parameter?)

Reply:
removed, will be changed in the future